plugins/ua_accesscontrol_default.h

```diff
@@ -5,6 +5,7 @@
 #define UA_ACCESSCONTROL_DEFAULT_H_
 
 #include "ua_server.h"
+#include "ua_types_generated.h"
```

Comment:
Is this really needed here?
Sometimes CLion adds includes even if the are not needed here...